lib/linguist/heuristics.yml

```diff
@@ -216,6 +216,13 @@ disambiguations:
     pattern: '^<\?(?:php)?'
   - language: SourcePawn
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
+  - language: NASL
+    pattern:
+    - '^\s*include\s*\(\s*(?:"|'')[\\/\w\-\.:\s]+(?:"|'')\s*\)\s*;'
+    - '^\s*(?:global|local)_var\s+(?:[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)(?:,\s*[\w\-]+(?:\s*=\s*[\w\-"'']+|)\s*)*+\s*;'
+    - '^\s*namespace\s+[\w\-]+\s*(?:$^)*\s*\{'
```

Comment:
Change `\s*(?:$^)*\s*\{` to just `\s*{`:

~~~diff
-'^\s*namespace\s+[\w\-]+\s*(?:$^)*\s*\{'
+'^\s*namespace\s+[\w\-]+\s*{'
~~~

`$` and `^` are zero-length matches, so repeating them is meaningless (also invalid in many regex engines). If you meant to match newlines, they'll already be matched by `\s*`.

Reply:
Thanks, I wasn't sure if the new line would be accounted for that way but I was able to test it to make sure this time. Changed in latest commit.